api/src/services/__tests__/meetingsService.ts

```diff
@@ -162,7 +167,106 @@ describe('meetingsService', () => {
         )
       ).toEqual({ id: meetingNoteId });
     });
-
+    it('should resolve to null if no meeting note id exists', async () => {
+      const meetingId = 2;
+      const principalId = 3;
+      const agendaOwningParticipantId = 4;
+      const noteText = '';
+      const sortOrder = 0;
+      mockQuery(
+        'select `id` from `participants` where `meeting_id` = ? and `principal_id` = ?',
+        [meetingId, principalId],
+        [{ id: 6 }]
+      );
+      mockQuery(
+        'insert into `meeting_notes` (`agenda_owning_participant_id`, `authoring_participant_id`, `note_text`, `sort_order`) values (?, ?, ?, ?)',
+        [agendaOwningParticipantId, 6, noteText, sortOrder],
+        []
+      );
+      expect(
+        await createMeetingNote(
+          meetingId,
+          principalId,
+          agendaOwningParticipantId,
+          noteText,
+          sortOrder
+        )
+      ).toEqual(null);
+    });
+    it('should return a meetingNote object from a given meeting note id', async () => {
```

Comment:
This test description doesn't seem to describe the test case. The test case calls `createMeetingNote` and asserts that a newly created meeting note is returned. There's already a test case for that on line [138](https://github.com/OpenTree-Education/rhizone-lms/pull/336/files#diff-685314a9340c322ec0afa4fcb0607baa7c735cf90e3892e67dbf72c54686eb80R138) above. Is this test redundant?

Reply:
Yes this test case is redundant after fixing the previous test case. This has now been removed.